<?xml version="1.0"?>
<launch>
    <group ns="dextra_laser">
        <include file="$(find tj2_rplidar)/launch/tj2_rplidar_a2.launch">
            <arg name="serial_port" value="/dev/serial/by-path/platform-3610000.xhci-usb-0:2.4:1.0-port0"/>
            <arg name="lidar_node_name" value="dextra_rplidar"/>
            <arg name="laser_frame_id" value="dextra_laser_link"/>
        </include>

        <node pkg="laser_filters" type="scan_to_scan_filter_chain" name="dextra_laser_filter">
            <rosparam command="load" file="$(find tj2_rplidar)/config/dextra_laser_filter.yaml" />
        </node>
    </group>
</launch>
